react/week4/github-search/src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import UserSearch from "./pages/UserSearch";
+
+ReactDOM.render(<UserSearch />, document.getElementById("root"));
```

Comment:
Try to create App.js file inside the src folder and render the UserSearch.js component there. And then render the App.js component inside the index.js, this is just good practice.